README.md

```diff
@@ -115,6 +121,36 @@ Below is an example of JSON format for a job.
 }
 ```
 
+#### Status
+
+Description of job status:
+
+```
+0 = FreeAsWind
+1 = OnJob
+2 = Cancelled
+3 = Delivered
+```
+
+**Note:** Job with a status `0` should not ever be sent to the API, if that
+happens it's a bug.
+
+#### Types
+
+Description of job types:
+
+```
+0 = Unknow
```

Comment:
This should be Unknown

Reply:
Fixed the typo but had typo in the commit message, force pushed..